packages/falcon-magento2-api/src/Magento2ApiBase.js

```diff
@@ -11,6 +11,8 @@ const isEmpty = require('lodash/isEmpty');
 
 const DEFAULT_KEY = '*';
 
+import { codes } from '@deity/falcon-errors';
```

Comment:
@deity-bram my mistake, I sorry, this package does not use es6, 

```suggestion
const { codes } = require('@deity/falcon-errors');
```

Reply:
@deity-bram please accept this suggestion, otherwise this package will throw an error